lib/fog/key_manager/openstack/requests/get_acl.rb

```diff
@@ -0,0 +1,67 @@
+module Fog
+  module KeyManager
+    class OpenStack
+      class Real
+        def get_secret_acl(uuid)
+          request(
+              :expects => [200],
+              :method  => 'GET',
+              :path    => "secrets/#{uuid}/acl",
+          )
+        end
+        
+	      def get_container_acl(uuid)
+          request(
+              :expects => [200],
+              :method  => 'GET',
+              :path    => "containers/#{uuid}/acl",
+          )
+        end
+
+      end
+
+      class Mock
+
+        def get_secret_acl(_uuid)
+          response = Excon::Response.new
+          response.status = 200
+          response.body = {
+              "data" => {
```

Comment:
Thanks for adding some mocks :) Would you please format them a bit more? And I believe the keys `data` and `body` are wrong here, i.e. not needed to wrap the response's body